src/test/java/seedu/us/among/logic/commands/EditCommandTest.java

```diff
@@ -48,25 +48,25 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    // @Test
-    // public void execute_someFieldsSpecifiedUnfilteredList_success() {
-    //     Index indexLastEndpoint = Index.fromOneBased(model.getFilteredEndpointList().size());
-    //     Endpoint lastEndpoint = model.getFilteredEndpointList().get(indexLastEndpoint.getZeroBased());
+    @Test
+    public void execute_someFieldsSpecifiedUnfilteredList_success() {
+        Index indexLastEndpoint = Index.fromOneBased(model.getFilteredEndpointList().size());
+        Endpoint lastEndpoint = model.getFilteredEndpointList().get(indexLastEndpoint.getZeroBased());
 
-    //     EndpointBuilder endpointInList = new EndpointBuilder(lastEndpoint);
-    //     Endpoint editedEndpoint = endpointInList.withMethod(VALID_METHOD_POST).withTags(VALID_TAG_CAT).build();
+        EndpointBuilder endpointInList = new EndpointBuilder(lastEndpoint);
+        Endpoint editedEndpoint = endpointInList.withMethod(VALID_METHOD_POST).withTags(VALID_TAG_CAT).build();
 
-    //     EditEndpointDescriptor descriptor = new EditEndpointDescriptorBuilder().withName(VALID_METHOD_POST)
-    //             .withTags(VALID_TAG_CAT).build();
-    //     EditCommand editCommand = new EditCommand(indexLastEndpoint, descriptor);
+        EditEndpointDescriptor descriptor = new EditEndpointDescriptorBuilder().withName(VALID_METHOD_POST)
+                .withTags(VALID_TAG_CAT).build();
+        EditCommand editCommand = new EditCommand(indexLastEndpoint, descriptor);
 
-    //     String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_ENDPOINT_SUCCESS, editedEndpoint);
+        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_ENDPOINT_SUCCESS, editedEndpoint);
 
-    //     Model expectedModel = new ModelManager(new EndpointList(model.getEndpointList()), new UserPrefs());
-    //     expectedModel.setEndpoint(lastEndpoint, editedEndpoint);
+        Model expectedModel = new ModelManager(new EndpointList(model.getEndpointList()), new UserPrefs());
+        expectedModel.setEndpoint(lastEndpoint, editedEndpoint);
 
-    //     assertCommandSuccess(editCommand, model, expectedMessage, expectedModel); //to-do
-    // }
+        assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
+    }
```

Comment:
Since we are not done with EditCommand yet, perhaps we should keep this as a to-do since when we change EditCommand it may lead to some more errors here.

Reply:
I think there's a good chance more tests will be breaking given how the `header` and `data` attributes have not been implemented yet. At the same time, I am also hesitant about commenting out all the test cases.

The consideration I have is that making several small fixes to pass them might be easier compared to having to make a single big change to all test cases at the same time. Depending on the implementation for `header` and `data`, ensuring the test cases pass now may prevent further complications down the road. Just a thought :D